portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,30 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> fruitFacts;
+
+  @Override
+  public void init() {
+    fruitFacts = new ArrayList<>();
+    fruitFacts.add("A strawberry is not an actual berry");
+    fruitFacts.add("Kiwis contain more Vitamin C than oranges");
+    fruitFacts.add("The pineapple is actually a berry");    
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Kiwis contain more Vitamin C than oranges :o");
+    // Convert the fruitFacts ArrayList to JSON
```

Comment:
nit: unneeded comment since the method name is self-explanatory. Maybe rename the variable to fruitFactsJson or something like that